### PR TITLE
Added Vec2i variants for ImagePainter methods

### PR DESCRIPTION
**Pull request by Maksasj**
Closing #40

### CODE REVIEW

File: src/bebone/assets/image/image_painter.cpp [hunk 1]
```diff
@@ -50,6 +69,17 @@ namespace bebone::assets {
         }
     }
 
+    template<typename Color>
+    void ImagePainter<Color>::paint_square(const Vec2i& pos, const Vec2i & size, const Color& color) {
```
Comment (Soskar1) [inline src/bebone/assets/image/image_painter.cpp:73]: Change `const Vec2i & size` to `const Vec2i& size`

File: src/bebone/assets/image/image_painter.h [hunk 2]
```diff
@@ -46,6 +61,14 @@ namespace bebone::assets {
             */
             void paint_square(const size_t& x_pos, const size_t& y_pos, const f32& sq_width, const f32& sq_height, const Color& color);
 
+            /*!
+            * Draws square to the image
+            * @param pos - Square bottom left corner position
+            * @param size - Square size
+            * @param color - Square color
+            */
+            void paint_square(const Vec2i& pos, const Vec2i & size, const Color& color);
```
Comment (Soskar1) [inline src/bebone/assets/image/image_painter.h:70]: Change `const Vec2i & size` to `const Vec2i& size`

**Review by Soskar1 — CHANGES_REQUESTED** [review]

File: src/bebone/assets/image/image_painter.h (hunk 2 above)
Comment (Maksasj) [reply src/bebone/assets/image/image_painter.h:70]: Ok boss !

File: src/bebone/assets/image/image_painter.cpp (hunk 1 above)
Comment (Maksasj) [reply src/bebone/assets/image/image_painter.cpp:73]: Ok boss !

**Review by Soskar1 — APPROVED** [review]

**Pull request merged**